frontend/app_flowy/lib/startup/tasks/app_widget.dart

```diff
@@ -73,7 +73,7 @@ class ApplicationWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     const ratio = 1.73;
     const minWidth = 600.0;
-    setWindowMinSize(const Size(minWidth, minWidth / ratio));
+    windowManager.setSize(const Size(minWidth, minWidth / ratio));
```

Comment:
The previous one is `setWindowMinSize` and now is `setSize`. We should keep the same logic as before.

Reply:
Previously (and now) we are just calling a library directly from a widget, so the choice of name is because of their implementation.

My suggestion would be to have a service that abstracts the implementation away. Then from outside the service we can call things predictably, like:

_Initialize on app startup_

```dart
AppWindow.instance.initialize();
```

_Show / hide to tray_

```dart
AppWindow.instance.hide();
```

```dart
AppWindow.instance.show();
```

etc.

Reply:
My bad, I misread; fixed that now. :+1: